src/main/scala/nl.knaw.dans.easy.authinfo/EasyAuthInfoApp.scala

```diff
@@ -66,21 +68,21 @@ trait EasyAuthInfoApp extends AutoCloseable with DebugEnhancedLogging with Appli
   }
 
   private def fromBagStore(bagId: UUID, path: Path): Try[Option[CachedAuthInfo]] = {
-    logger.info(s"[$bagId] item for path $path not found in autCache, trying to retrieve item from bagStore")
+    logger.info(s"[$bagId] item for path ${path.escapePath} not found in authCache, trying to retrieve item from bagStore")
```

Comment:
@rkruithof could you move this comment to line 38, above. Seems more appropriate and consistent with the other cases to put the logging there.